lib/truncate.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = exports = {};
+
+  // Open file, truncate data to first 8 bytes, print result in hex form
+
+exports.truncatePoem = function(poem) {
+  fs.open(poem, 'r+', function(err, fileDesc) {
```

Comment:
Super stoked to see you reading the docs and finding creative solutions to problems like this!